OpenRA.Mods.Common/Scripting/Properties/ProductionProperties.cs

```diff
@@ -34,19 +34,22 @@ public ProductionProperties(ScriptContext context, Actor self)
 
 		[ScriptActorPropertyActivity]
 		[Desc("Build a unit, ignoring the production queue. The activity will wait if the exit is blocked.",
-			"If productionType is nil or unavailable, then an exit will be selected based on Buildable info.")]
+			"If productionType is nil or unavailable, then an exit will be selected based on 'Buildable.BuildAtProductionType'.",
+			"If 'Buildable.BuildAtProductionType' is not set either, a random exit will be selected.")]
```

Comment:
Where does this choose a random exit?

Reply:
https://github.com/OpenRA/OpenRA/blob/bleed/OpenRA.Mods.Common/Traits/Production.cs#L116
https://github.com/OpenRA/OpenRA/blob/bleed/OpenRA.Mods.Common/Traits/Production.cs#L102